examples/resources/netbox_config_template/resource.tf

```diff
@@ -0,0 +1,6 @@
+resource "netbox_config_template" "test" {
+	name = "test"
+	description = "test description"
+	template_code = "hostname {{ name }}"
+	environment_params = jsonencode({"name" = "my-hostname"})
+}
```

Comment:
If you run `terraform fmt -recursive`, the tabs here should be reformatted with spaces (among other things). Then make docs.